bin/showmehow-service.js.in

```diff
@@ -29,6 +29,12 @@ const Controller = imports.lib.controller;
 const Descriptors = imports.lib.descriptors;
 const Service = imports.lib.service;
 
+// HOME might not be set if we're running through D-Bus activation, so
+// set the environment variable to the value of GLib.get_home_dir which will
+// fall back to passwd
+if (!GLib.getenv('HOME')) {
+    GLib.setenv('HOME', GLib.get_home_dir());
```

Comment:
Not sure if this will work, since `GLib.setenv()` takes three parameters; the third, `overwrite` controls whether an existing value should be overwritten, so you could actually just pass `false` and avoid the conditional.